tests/generic_config_updater/test_eth_interface.py

```diff
@@ -144,6 +144,9 @@ def test_toggle_pfc_asym(duthost, ensure_dut_readiness, pfc_asym):
 
 @pytest.mark.parametrize("fec", ["rs", "fc"])
 def test_replace_fec(duthost, ensure_dut_readiness, fec):
+    if duthost.facts['platform'] == 'x86_64-kvm_x86_64-r0':
+        pytest.skip("Not supported on virtual switch")
```

Comment:
Is it better to skip using the conditional mark?

Reply:
@wangxin could you share an example?

Reply:
Conditional mark is a plugin for adding marks based on some customized conditions. For example, add `skip` mark if platform is `x86_64-kvm_x86_64-r0`. By using this plugin, tests can be skipped in earlier stage to save some overhead of preparing fixtures.
Document is here: https://github.com/Azure/sonic-mgmt/tree/master/tests/common/plugins/conditional_mark
Examples are here: https://github.com/Azure/sonic-mgmt/blob/master/tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

Please ensure that the added entries are in alphabetic order, so it is easier for people to read the code and the chance of adding duplicated entries is low.

Reply:
thank you, I changed to conditional mark